alpine/aarch64/3.5/entry.sh

```diff
@@ -44,6 +44,12 @@ function init_openrc()
 	printf '%q ' "$@" >> /etc/resinApp.sh
 	chmod +x /etc/resinApp.sh
 
+	# Make sure there is no pidfile or process running before starting.
+	if [ -f "/var/run/resinapp.pid" ]; then
+		kill $(cat /var/run/resinapp.pid) &> /dev/null
```

Comment:
You don't have to try and kill this. The problem was most likely caused by a container restart where the pid file persisted in the writable layer. So you can just leave the rm here

Reply:
Updated!